wincallback.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+
+// Generate Windows callback assembly file.
```

Comment:
This looks PR looks good. I think we should add wincallback.go to the README.md where it lists external Go files

Reply:
Done